[ERR] bulk flag is not supported with stdin

use notify with github action 


```
      - name: Notify Reports
        run: |
          for file in output/vul-${{ steps.date.outputs.date }}/nuclei/* ; do notify -provider-config config/provider-config.yaml -bulk -id dd -data $file; done
        shell: bash

```
```


Run for file in output/vul-20210908-0530/nuclei/* ; do notify -provider-config config/provider-config.yaml -bulk -id dd -data $file; done

             __  _ ___    
  ___  ___  / /_(_) _/_ __
 / _ \/ _ \/ __/ / _/ // /
/_//_/\___/\__/_/_/ \_, / v1.0.0
                   /___/  

		projectdiscovery.io

Use with caution. You are responsible for your actions
Developers assume no liability and are not responsible for any misuse or damage.
[ERR] bulk flag is not supported with stdin
Error: Process completed with exit code 1.

```

i've been tested on terminal , everything is fine


![image](https://user-images.githubusercontent.com/16974004/132517593-104cad12-0630-4097-a6ee-410fa832d1b1.png)


but github action got   [ERR] bulk flag is not supported with stdin

--- a/internal/runner/runner.go
+++ b/internal/runner/runner.go
@@ -79,18 +79,17 @@ func (r *Runner) Run() error {
 	var err error
 
 	switch {
+	case r.options.Data != "":
+		inFile, err = os.Open(r.options.Data)
+		if err != nil {
+			gologger.Fatal().Msgf("%s\n", err)
+		}
 	case hasStdin():
 		if r.options.Bulk {
 			gologger.Error().Msgf("bulk flag is not supported with stdin")
 			os.Exit(1)
 		}
 		inFile = os.Stdin
-
-	case r.options.Data != "":
-		inFile, err = os.Open(r.options.Data)
-		if err != nil {
-			gologger.Fatal().Msgf("%s\n", err)
-		}
 	default:
 		return errors.New("notify works with stdin or file using -data flag")
 	}

